auditbeat/module/audit/file/_meta/docs.asciidoc

```diff
@@ -0,0 +1,42 @@
+=== Audit file metricset
+
+experimental[]
+
+The `file` metricset sends events when a file is changed (created, updated, or
+deleted) on disk. The events contain file metadata and hashes (MD5, SHA1, and
+SHA256).
+
+[float]
+=== How it works
+
+This metricset...
+
+[float]
+=== Configuration options
+
+This metricset has some configuration options for tuning its behavior. The
+following example shows all configuration options with their default values.
+
+[source,yaml]
+----
+- module: audit
+  metricsets: [file]
+  file.paths:
+    binaries:
+    - /bin
+    - /usr/bin
+    - /sbin
+    - /usr/sbin
+    conf:
+    - /etc
+----
+
+
+//REVIEWERS: Should the following desc say glob-based paths?
+
+*`file.paths.binaries`*:: A list of paths containing binary files to monitor for
```

Comment:
@andrewkroh I think this line might be misleading. Based on the other config you submitted, I'm guessing that last part of the config name (`binaries` or `conf`) can be any string, no?